content/php/concepts/math-functions/terms/ceil/ceil.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'ceil()'
+Description: 'Returns the next highest integer value of the fractional argument.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`ceil()`** function returns the next highest integer value of the fractional argument.
+
+## Syntax
+
+```pseudo
+$result = ceil(n);
+```
+
+Where `n` is the fractional number to find the next highest integer value of.
+
+## Example
```

Comment:
Generally we want a sentence describing what the example does, and output shown not in comments, but in a separate ` ```shell ` block afterwards (introduced by a sentence like, "This example results in the following output").